app/controllers/api/functions.php

```diff
@@ -1023,6 +1023,16 @@
         $results = $dbForProject->find('executions', $queries, $limit, $offset, [], [Database::ORDER_DESC], $cursorExecution ?? null, $cursorDirection);
         $total = $dbForProject->count('executions', $queries, APP_LIMIT_COUNT);
 
+        $roles = Authorization::getRoles();
+        $isPrivilegedUser = Auth::isPrivilegedUser($roles);
+        $isAppUser = Auth::isAppUser($roles);
+        if ($isPrivilegedUser || $isAppUser) {
```

Comment:
I think the condition here should be opposite. I think this part is important enough for having its own tests.

Reply:
Yes.. 👍